art/attacks/evasion/saliency_map.py

```diff
@@ -109,8 +109,9 @@ def generate(self, x: np.ndarray, y: Optional[np.ndarray] = None, **kwargs) -> n
 
             # Main algorithm for each batch
             # Initialize the search space; optimize to remove features that can't be changed
-            search_space = np.zeros(batch.shape)
+            search_space = np.ones(batch.shape)
```

Comment:
Could you please move this line into an `else` block?

Reply:
Sure ! Thank you for the review.